<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>*********************************************************************
*********************************************************************</comment>
 <class>SlnForm</class>
 <widget class="QWidget" name="SlnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;Project&lt;/b&gt;&lt;p&gt;Choose the topic you want from the contents list.&lt;/p&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="whatsThis">
        <string>Displays contents of project</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pageContent">
        <property name="whatsThis">
         <string>&lt;b&gt;Project&lt;/b&gt;&lt;p&gt;Choose the topic you want from the contents list.&lt;/p&gt;</string>
        </property>
        <attribute name="title">
         <string>Contents</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="SlnTreeWidget" name="treeContents">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>&lt;b&gt;Contents list of project.&lt;/b&gt;&lt;p&gt;Click an item to see the page.&lt;/p&gt;</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSearch">
        <attribute name="title">
         <string>Search</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="4" colspan="4">
          <widget class="QPushButton" name="pushOptions">
           <property name="text">
            <string>Options</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="8">
          <widget class="QLineEdit" name="lineSearch"/>
         </item>
         <item row="11" column="0" colspan="8">
          <widget class="QTreeWidget" name="treeResults">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QPushButton" name="pushSearch">
           <property name="text">
            <string>Search all</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QPushButton" name="pushFindPrev">
           <property name="text">
            <string>Find back</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" colspan="4">
          <widget class="QPushButton" name="pushFindNext">
           <property name="text">
            <string>Find next</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="7">
          <widget class="QLineEdit" name="lineNode">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QPushButton" name="pushSelNode">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageFavorites">
        <attribute name="title">
         <string>Favorites</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboFavRoot"/>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeFavorites">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlnTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>./src/gui/slntreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeContents</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
